posthog/hogql/grammar/HogQLParser.g4

```diff
@@ -167,6 +169,14 @@ columnLambdaExpr:
     ARROW columnExpr
     ;
 
+
+tagElement     :   LT identifier tagAttribute* SLASH GT;
+
+tagAttribute   :   identifier '=' STRING_LITERAL
+               |   identifier '=' LBRACE columnExpr RBRACE
+               |   identifier
+               ;
```

Comment:
Given this is an embedded language in a way, I think it's worth using the same naming convention as with the AST nodes, i.e. prefixing with "HogQLX" (`hogqlxTagElement`, `hogqlxTagAttribute`)